index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>My First Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+
+<body>
+<!-- first card box -->
+  <article class="firstcard">
+    <header class="top"><h1>Dull Playing Cards</h1></header>
+    <section>
+      <img src="images/cards1.jpg">
+      <p>These cards are dull and worn. Broken in. They come in colors. There are 52 of them. They feel good in your hand. </p>
+      <div class="available">Available</div>
+    </section>
+
+    <section>
+      <header class="subheading">Specifications</header>
+      <div class="block">Size: 2" x 3"</div>
+      <div class="block">Weight: 0.5 lb</div>
```

Comment:
So divs are already block level elements, you don't need to give them a class just to re-add the display block. it doesn't hurt anything, but it isn't needed. ;)